examples/run/cpu/truth_fitting_example.cpp

```diff
@@ -114,9 +114,12 @@ int main(int argc, char* argv[]) {
         0.01 / detray::unit<scalar>::GeV,
         1 * detray::unit<scalar>::ns};
 
+    // Propagation configuration
+    detray::propagation::config propagation_config(propagation_opts);
+
     // Fitting algorithm object
     typename traccc::fitting_algorithm<host_fitter_type>::config_type fit_cfg;
-    fit_cfg.propagation = propagation_opts.config;
+    fit_cfg.propagation = propagation_config;
```

Comment:
Since in this example `detray::propagation::config` is only needed once, creating a separate `propagation_config` object is not really needed here, no? 🤔